commands/orchestrationcommands/stackeventcommands/get.go

```diff
@@ -0,0 +1,110 @@
+package stackeventcommands
+
+import (
+	"github.com/rackspace/rack/commandoptions"
+	"github.com/rackspace/rack/commands/orchestrationcommands/stackcommands"
+	"github.com/rackspace/rack/handler"
+	"github.com/rackspace/rack/internal/github.com/codegangsta/cli"
+	"github.com/rackspace/rack/internal/github.com/rackspace/gophercloud/rackspace/orchestration/v1/stackevents"
+	"github.com/rackspace/rack/util"
+)
+
+var get = cli.Command{
+	Name:        "get",
+	Usage:       util.Usage(commandPrefix, "get", "[--name <stackName> | --id <stackID>] --resource <resourceName> --event <eventID>"),
```

Comment:
Because we're talking about events now:
"[--stack-name <stackName> | --stack-id <stackID>] --resource <resourceName> --id <eventID>"